src/pages/List/List.scss

```
@@ -58,8 +58,8 @@
       .classTab {
         display: flex;
         flex-direction: column;
-        align-items: flex-start;
-        width: 900px;
+        align-items: center;
+        width: 1000px;
```

width값을 임의의 값으로 고정시키는 것은 반응형 웹을 고려했을때 의도치 않은 UI 깨짐 현상을 가져올 수 있습니다.
최초 혹은 최소 1000px의 넓이는 가지고있어야 하는 UI라면
```suggestion
        width : 100%;
        max-width: 1000px;
```
예시코드와 같은 방식으로 기본적으로 해당 UI가 가져야 할 넓이와 반응형 UI를 대처 할 수 있습니다.